### ISSUE TITLE
Concordance for prognostic models with competing risks

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#2]
## TL;DR
This paper proposes a define a cause-specific concordance index in the presence of competing risks.

### Paper Link

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4059461/

### Author/Institution

Wolbers M (Oxford University), Blanche P (Université Bordeaux Segalen), Koller MT (University Hospital Basel), Witteman JC (Erasmus Medical Center), Gerds TA (University of Copenhagen)

## Overview

Several measures for quantifying the accuracy of prognostic models have been adapted from the standard survival setting with only one failure cause to competing risks. For survival data, the concordance index is a frequently reported measure of discrimination and we have previously presented a simple adaptation of Harrell's concordance estimator to the competing risks setting.

## Methods

Survival analysis

## Cite

Wolbers M, Blanche P, Koller MT, Witteman JC, Gerds TA. Concordance for prognostic models with competing risks. Biostatistics. 2014 Jul;15(3):526-39. doi: 10.1093/biostatistics/kxt059. Epub 2014 Feb 2. PMID: 24493091; PMCID: PMC4059461.

## Comments

R code available
